Automattic/wp-calypso
# chore: Update encrypted secrets entry for simpleSiteFreePlanUser.

#### Proposed Changes

This PR updates the entry for the `simpleSiteFreePlanUser` entry. This is to support https://github.com/Automattic/wp-calypso/pull/71333 which requires the use of a newer test site (created after 2022/12/15).

Key changes:
- set a new test site as primary.
- retain the old test site as secondary test site.

#### Testing Instructions

Ensure the following build configurations are passing:

If further testing is required, download and decrypt the [encrypted config file](packages/calypso-e2e/src/secrets/encrypted.enc).

#### Pre-merge Checklist





Related to #
